Rename periodic para plugin

quanru opened the pull request:
1. The current plugin name can no longer cover existing functions.
2. Prepare for adding more features later.
3. Fits with the existing official website: https://obsidian-life-os.netlify.app/

joethei commented:
Remove "Obsidian" from the description, and remove our logo from your website.

Also, naming something "Obsidian xyz" is reserved for things we create.
You are free to call it "LifeOS for Obsidian", but not "Obsidian LifeOS"

quanru commented:
@joethei I have made the modifications according to your requirements, please have a look.

1. removed "Obsidian" from the description
2. removed obsidian logo from my website
3. change all "Obsidian LifeOS" to "LifeOS for Obsidian"
